app/models/price_table.rb

```diff
@@ -1,5 +1,5 @@
 class PriceTable < ApplicationRecord
-    validates_presence_of :trainees, :pricePoint
+    validates_presence_of :trainees, :price_point
     has_many :sessions
-    enum pricePoint: { low: 0, medium: 1, high: 2 }
+    enum price_point: { low: 0, medium: 1, high: 2 }
```

Comment:
Why did you change the variable name to snake case again?

Reply:
It's  a rails db column, rails convention to use snake case as we did with every other db column in our db. 
Also the tests were failing

Reply:
Okay I see!